src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.Scanner;
+
 public class Duke {
+    
+    public static final String DIVIDER = "____________________________________________________________\n";
+    
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        greet();
+        
+        Tasks list = new Tasks();
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+        
+        while (!line.equalsIgnoreCase("bye")) {
+            System.out.print(DIVIDER);
+            runCommand(list, line);
+            System.out.println(DIVIDER);
+            line = in.nextLine();
+        }
+        exit();
+    }
```

Comment:
Maybe adding a line here would be good as there's no space between these two methods.